.github/workflows/lynx.yml

```diff
@@ -0,0 +1,52 @@
+name: '🔎 Lynx Testnet Example'
+
+on:
+  schedule:
+    - cron: "0/20 * * * *"  # Every 20 minutes
+  pull_request:
+    # branches:
+    #   - main
+    #   - development
+    #   - epic-*
```

Comment:
Is this intentional? Does this mean for any PR then?

Reply:
Yes, at least for the moment since we're working on the monorepo branch. But I also think that what's the problem about running this for every PR?

Reply:
No real issue - just wondering if this was commented out on purpose.